package.json

```diff
@@ -96,7 +96,7 @@
     "@formatjs/intl-pluralrules": "^4.1.3",
     "@formatjs/intl-relativetimeformat": "^9.2.3",
     "@ledgerhq/react-native-hw-transport-ble": "6.27.3",
-    "@react-native-async-storage/async-storage": "1.15.5",
+    "@react-native-async-storage/async-storage": "^1.17.11",
```

Comment:
is it required for this PR ?

Reply:
dunno, that was part of the init script

Reply:
does this break anything?

Reply:
minor version, not supposed to, just curious why is it bumping it 🤔